Assignment3Code/src/assignment3code/Assignment3Code.java

```diff
@@ -0,0 +1,47 @@
+
+
+package assignment3code;
+import java.util.Scanner;
+import javax.swing.JOptionPane; //for dialog box to appear throughout program 
+ 
+
+public class Assignment3Code {
+
+  
+    public static void main(String[] args) {
+       //dec of all variables
+        Scanner keyboard = new Scanner (System.in);
+        String wordToReverse;
+        String reverse = "";
+
+        //obtaining the input
+      
+       
+        wordToReverse = JOptionPane.showInputDialog("Enter a word with atleast five"
+                + " characters.");
+        //incase user enters string less than 5 letters
+        while (wordToReverse.length() <5){
```

Comment:
Good job doing error checking!  I didn't even think to do this for mine to check for the length before allowing the character swap to occur.